modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1012,6 +1011,8 @@ public void dropTable(String name) {
                     }
                 }
             } catch (Throwable t) {
+                assert false;
```

Comment:
what is this?

Reply:
I guess that was the reminder for myself. That exception handling is a complete nonsense.